packages/builder/src/builtin-tasks/clean.ts

```diff
@@ -1,16 +0,0 @@
-import fsExtra from "fs-extra";
-
-import { task } from "../internal/core/config/config-env";
-
-import { TASK_CLEAN } from "./task-names";
-
-export default function () {
-  task(
-    TASK_CLEAN,
-    "Clears the cache and deletes all artifacts",
-    async (_, { config }) => {
-      await fsExtra.remove(config.paths.cache);
-      await fsExtra.remove(config.paths.artifacts);
-    }
-  );
-}
```

Comment:
why removing this file?

Reply:
I added this file as a task and currently we don't know what the project structure will be so command `algob cleanup` doesn't make sense.

Reply:
It might be useful. We defined a project structure last week during a meeting. The artifacts are in `/artifacts/
https://paper.dropbox.com/doc/Algorand-builder-architecture--A2ESMswpL3US1ZltRo1W~J7wAg-Vcdp0XNngizChyUWvFXfs

Reply:
- [ ] Restore